tanner/config.py

```diff
@@ -3,22 +3,23 @@
 import os
 import sys
 
+TANNER_DIR = os.path.expanduser("~/.tanner")
```

Comment:
And what if I want to setup TANNER_DIR? I should always change it in the code?

Reply:
we can have an option while setup to change the default directory.........should we do this? is there a need for this?

Reply:
if it's possible and easy to do, I definitely prefer to have a control on folders usage :)